### PR TITLE
AWS actions linting update

### PR DESCRIPTION
**Pull request by ggabernet**
- Updated linting for the GitHub actions AWS workflows.
- Also updated linting for the Json schema after the change of `https` to `http` by #748 

## PR checklist

 - [x] This comment contains a description of changes (with reason)
 - [x] `CHANGELOG.md` is updated
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Documentation in `docs` is updated

### CODE REVIEW

**Review by ewels — APPROVED** [review]
Nice! Thanks for fixing the schema tests - not sure why that didn't come up as failing for me on that PR?

LGTM, though only on my phone so tricky to check properly.

**Comment by ggabernet** [comment]
yeah I also find it weird that it did not come up, but may be because it was only a warning?

Sure, I can also wait until you are back to a laptop to merge if you want to double check, no rush!

File: nf_core/lint.py
Comment (ewels) [inline nf_core/lint.py:853]: Super minor: Clearer to read if the `is` / `is not` logic is grouped together..

```suggestion
                assert "workflow_dispatch" in wf[True]
                assert "push" not in wf[True]
                assert "pull_request" not in wf[True]
```

**Review by ewels — APPROVED** [review]
Love it 🚀

**Pull request merged**
